nirc_ehr/src/org/labkey/nirc_ehr/table/NIRC_EHRCustomizer.java

```diff
@@ -191,6 +192,20 @@ private void customizeAnimalTable(AbstractTableInfo ds)
             col15.setDescription("Shows the total offspring of each animal");
             ds.addColumn(col15);
         }
+        if (ds.getColumn("flags") == null)
+        {
+            var col = getWrappedCol(us, ds, "Flags", "demographicsActiveFlags", "Id", "Id");
+            col.setLabel("ActiveFlags");
+            col.setURL(DetailsURL.fromString("/query/executeQuery.view?schemaName=ehr_lookups&queryName=flag_values&query.Id~eq=${Id}", ds.getContainerContext()));
+            ds.addColumn(col);
+        }
+        if (ds.getColumn("demographicsActiveAssignment") == null)
+        {
+            var col21 = getWrappedCol(us, ds, "activeAssignments", "demographicsActiveAssignment", "Id", "Id");
+            col21.setLabel("Assignments - Active");
```

Comment:
Should we be consistent in terms of "Active X" vs "X - Active"?

Reply:
Yeah and good to label it as Protocol Assignment since they'll have both protocol and projects